tests/pc/test_po_cleanup.py

```diff
@@ -27,36 +27,38 @@ def ignore_expected_loganalyzer_exceptions(rand_one_dut_hostname, loganalyzer):
         ]
         loganalyzer[rand_one_dut_hostname].ignore_regex.extend(ignoreRegex)
         expectRegex = [
-            ".*teamd#teammgrd: :- cleanTeamProcesses.*",
-            ".*teamd#teamsyncd: :- cleanTeamSync.*"
+            ".*teammgrd: :- cleanTeamProcesses.*",
+            ".*teamsyncd: :- cleanTeamSync.*"
         ]
         loganalyzer[rand_one_dut_hostname].expect_regex.extend(expectRegex)
 
-
-def check_kernel_po_interface_cleaned(duthost):
-    res = duthost.shell("ip link show | grep -c PortChannel",  module_ignore_errors=True)["stdout_lines"][0].decode("utf-8")
+def check_kernel_po_interface_cleaned(duthost, asic_index):
+    namespace = duthost.get_namespace_from_asic_id(asic_index)
+    res = duthost.shell(duthost.get_linux_ip_cmd_for_namespace("ip link show | grep -c PortChannel", namespace),module_ignore_errors=True)["stdout_lines"][0].decode("utf-8")
     return res == '0'
 
-
-def test_po_cleanup(duthosts, rand_one_dut_hostname, tbinfo):
-    """
-    test port channel are cleaned up correctly and teammgrd and teamsyncd process
-    handle  SIGTERM gracefully
-    """
+@pytest.fixture(scope="module", autouse=True)
+def disable_teamd(duthosts, rand_one_dut_hostname, tbinfo):
```

Comment:
Not sure I understood this. How is the teamd being disabled. From your description looks like you want to stop "swss" instead of "teamd". Is this fixture used only to skip the test and perform config reload.

Reply:
@smaheshm Updated Function name